functionaltests/TestUpgrade_8_15_4_to_8_16_0/main.tf

```diff
@@ -0,0 +1,74 @@
+terraform {
+  required_version = ">= 0.12.29"
+
+  required_providers {
+    ec = {
+      source  = "elastic/ec"
+      version = "0.12.2"
+    }
+  }
+}
+
+variable "name" {
+  type        = string
+  description = "The deployment name"
+}
+
+variable "stack_version" {
+  type        = string
+  description = "The Elasticsearch version to bootstrap"
+}
+
+provider "ec" {
+  endpoint = "https://public-api.qa.cld.elstc.co"
+}
+
+resource "ec_deployment" "example_minimal" {
```

Comment:
Can you add some labels so we know those deployments/projects are coming from this automation?

Reply:
https://github.com/elastic/apm-server/blob/23d9531df96c72b31bbf062475cf852a337f7fbb/testing/benchmark/main.tf#L21-L37

and

https://github.com/elastic/apm-server/blob/23d9531df96c72b31bbf062475cf852a337f7fbb/testing/benchmark/main.tf#L114


can help with

Reply:
Thanks for reminding me :pray: Added in [e56e937](https://github.com/elastic/apm-server/pull/14935/commits/e56e9374e1a42e9a2ce58b558209f7ca7179cdab)